stable_baselines3/common/torch_layers.py

```diff
@@ -224,6 +228,88 @@ def forward(self, features: th.Tensor) -> Tuple[th.Tensor, th.Tensor]:
         return self.policy_net(shared_latent), self.value_net(shared_latent)
 
 
+class CombinedExtractor(BaseFeaturesExtractor):
+    """
+    Combined feature extractor for Dict observation spaces.
+    Builds a feature extractor for each key of the space. Input from each space
+    is fed through a separate submodule (CNN or MLP, depending on input shape),
+    the output features are concatenated and fed through additional MLP network ("combined").
+
+    :param observation_space:
+    :param features_dim: Number of features extracted.
+        This corresponds to the number of unit for the last layer.
+    :param cnn_output_dim: Number of features to output from each CNN submodule(s)
+    :param mlp_output_dim: Number of features to output from each MLP submodule(s)
+    :param mlp_net_arch: Architecture of each MLP network module
+    :param activation_fn: The activation function used in all MLP submodules and combined network
+    :param combined_net_arch: Architecture of the combined network module which calculates the final feature extracted
+    """
+
+    def __init__(
+        self,
+        observation_space: gym.spaces.Dict,
+        features_dim: int = 64,
+        cnn_output_dim: int = 64,
+        mlp_output_dim: int = 64,
+        mlp_net_arch: List[int] = [64, 64],
```

Comment:
please avoid mutables as default, see https://github.com/satwikkansal/wtfpython#-beware-of-default-mutable-arguments